index.html

```diff
@@ -72,7 +72,67 @@ <h3 class="subtitle">Software developer</h3>
             <a href="#"><img src="./img/icons/medium.svg" alt="github-icon" /></a>
           </li>
         </ul>
+        <div class="rel-links">
+          <a href="#"><img id="alt-img" src="./img/icons/down-arrow.png" alt="down-arrow-icon" /></a>
+          <a href="#"><img src="./img/icons/Work.svg" alt="work-icon" /></a>
+        </div>
       </div>
+      <section class="grid-section">
+        <h2 class="title alt-title">My Recent Works</h2>
+        <hr />
+        <section class="card-section">
+          <div class="cards top-card">
+            <img src="./img/linting-tools.webp" alt="linters-image" />
+            <div class="card-txt">
+              <h3>HTML/CSS linters</h3>
+              <p>A linters project for HTML and CSS.</p>
+              <p>
+                Linters help to correct common errors that are hard to detect on the code base. It helps to enforce some level of professionality and code
+                standard for better optimization and readability
+              </p>
+              <div class="card-footer">
+                <ul class="card-footer-list alt-menu">
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">html</a>
+                  </li>
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">css</a>
+                  </li>
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">github</a>
+                  </li>
+                </ul>
+                <button class="btn" type="button">See project</button>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="card-section">
+          <div class="cards alt-card">
+            <img src="./img/html-css.jpg" alt="html/css-image" />
+            <div class="card-txt">
+              <h3>A personal portfolio project built with HTML and CSS.</h3>
+              <p>
+                Professional portfolio site built with HTML/CSS with standard githubflow. Built to satisfy professional standards of designs and adaptability.
+              </p>
+              <div class="card-footer">
+                <ul class="card-footer-list alt-menu">
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">html</a>
+                  </li>
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">css</a>
+                  </li>
+                  <li class="card-footer-listItem">
+                    <a href="#" class="card-footer-links">github</a>
+                  </li>
+                </ul>
+                <button class="btn" type="button">See project</button>
+              </div>
```

Comment:
- [ ] As suggested by the previous reviewer, your buttons should follow the styling design of user actions like `:hover`, `:active`, `:disabled`. KIndly fix that to stick to the Figma design. In your case, you just added a box shadow on the hover effect but there are other actions. Note that the colors change too.

**This is how they should on each action:**

![image](https://user-images.githubusercontent.com/80612925/176761909-e252c8b3-16fb-4650-a4f0-437dca94ef94.png)